language/move-prover/bytecode-to-boogie/tests/translator_tests.rs

```diff
@@ -52,87 +51,61 @@ fn test3() {
     let written_code = fs::read_to_string("src/bytecode_instrs.bpl").unwrap();
     res.push_str(&written_code);
     res.push_str(&ts.translate());
-    // This is probably too sensitive to minor changes in libra; commenting for now.
-    //    let expected_code = fs::read_to_string("test_mvir/test3.bpl.expect").unwrap();
-    //    assert_eq!(res, expected_code);
+    res
 }
 
-#[test]
-fn test_arithmetic() {
-    let mut file_names = vec![];
-    let name = "test_mvir/test-arithmetic.mvir".to_string();
-    file_names.push(name);
-    let (modules, source_maps) = compile_files(file_names.to_vec());
+fn run_boogie(boogie_str: String) {
```

Comment:
```suggestion
fn run_boogie(boogie_str: &str) {
```

Will make this nicer for callers passing literals

Reply:
Done.